### ISSUE TITLE
webdriver-manager --update uses unreleased chrome driver versions

### ISSUE THREAD

**Issue by fippo** [angular/webdriver-manager#304]
it seems that webdriver-manager --update pulls yet unreleased versions of chromedriver. Currently that is 2.37 which according to https://sites.google.com/a/chromium.org/chromedriver/ is not the latest.

https://chromedriver.storage.googleapis.com/index.html?path=2.37/
exists and has a linux binary.
https://chromedriver.storage.googleapis.com/index.html?path=2.36/
also has a notes.txt which I assume are the release notes which only exist for releases

**Comment by BorisOsipov** [comment]
Hi @cnishina!

I am a commiter of [selenide](https://github.com/codeborne/selenide) and [webdriverio](https://github.com/webdriverio/webdriverio/) and would like to have some advice from you regarding selecting latest version of browser binaries.

As I see in webdriver-manager for js and [java implemetation ](https://github.com/bonigarcia/webdrivermanager/) it is common practice to select the latest version by manual parsing xml files instead of using versions from LATEST_RELEASE.

Do you now some hidden disadvantages(or maybe some negative experience) to not use LATEST_RELEASE prior to parse xml?
Your experience will help me. Hope you will have time to help me.

Thanks in advance!

**Comment by cnishina** [comment]
@fippo Is this still an issue? Is the concern that the ChromedDriver didn't work with the Chrome version? Potentially the website was not updated in time for the Chrome version.

@BorisOsipov 

Here's some history about this: We at one point maintained a version of binaries. Part of our release process was to update the configuration file to download the version we supported. After the gecko driver became stable with the FireFox versions and selenium-webdriver 3 was released, we were able to always grab the latest.  So far downloading the xml or json files hasn't been a problem. We also allow users to download a specific version via a flag.

I am trying to do a rewrite with a few lessons learned, less OO spaghetti (my personal opinion about my own programming / decision making), and more unit / integration testing on my free time. I hope to make it a library rather than just a CLI and to reach some feature parity in the next few months. Here's my work in progress (https://github.com/cnishina/webderpy-manager). Is this something that might interest you for selenide or webdriverio?
